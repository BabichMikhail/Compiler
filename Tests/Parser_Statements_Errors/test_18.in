begin while 1 do else end.
