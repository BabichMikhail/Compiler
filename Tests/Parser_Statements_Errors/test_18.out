Fatal: Syntax error, "end" expected but "else" found in Line 1 Column 21
